### ISSUE TITLE
Calling unknown method: craft\elements\Entry::getTitle()

### ISSUE THREAD

**Issue by boboldehampsink** [verbb/hyper#199]
### Describe the bug

On Hyper 2.1.0 I see the following error:

```
yii\base\UnknownMethodException: Calling unknown method: craft\elements\Entry::getTitle() in /app/user/vendor/yiisoft/yii2/base/Component.php:308
Stack trace:
#0 /app/user/vendor/craftcms/cms/src/base/Element.php(2372): yii\base\Component->__call('getTitle', Array)
#1 /app/user/vendor/verbb/hyper/src/base/ElementLink.php(221): craft\base\Element->__call('getTitle', Array)
#2 /app/user/vendor/verbb/hyper/src/base/Link.php(483): verbb\hyper\base\ElementLink->getLinkText()
#3 /app/user/vendor/yiisoft/yii2/base/Component.php(139): verbb\hyper\base\Link->getText()
#4 /app/user/vendor/craftcms/cms/src/base/Element.php(2337): yii\base\Component->__get('text')
#5 /app/user/vendor/craftcms/cms/src/helpers/Template.php(137): craft\base\Element->__get('text')
#6 /tmp/storage/runtime/compiled_templates/fc/fcd24bc7cb21092f4b89989742dd7290.php(147): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\hyper\links\Entry), 'text', Array, 'any', false, false, false, 52)
#7 /app/user/vendor/twig/twig/src/Template.php(393): __TwigTemplate_26d775af4600cac7dff64ca2d4d0dc9d->doDisplay(Array, Array)
#8 /tmp/storage/runtime/compiled_templates/74/7467b9498591f30fc4b77cb15f1f8206.php(75): Twig\Template->yield(Array)
#9 /app/user/vendor/twig/twig/src/Template.php(437): __TwigTemplate_27a50d20589a8df34435d0a71d4438c4->block_hero(Array, Array)
#10 /tmp/storage/runtime/compiled_templates/09/099aea1042afbf0528769d5e1edddcf0.php(74): Twig\Template->yieldBlock('hero', Array, Array)
#11 /app/user/vendor/twig/twig/src/Template.php(437): __TwigTemplate_93328f652dcac9c5548e1a67014fa730->block_content(Array, Array)
#12 /app/user/vendor/twig/twig/src/Template.php(482): Twig\Template->yieldBlock('content', Array, Array, false)
#13 /tmp/storage/runtime/compiled_templates/74/7467b9498591f30fc4b77cb15f1f8206.php(105): Twig\Template->yieldParentBlock('content', Array, Array)
#14 /app/user/vendor/twig/twig/src/Template.php(437): __TwigTemplate_27a50d20589a8df34435d0a71d4438c4->block_content(Array, Array)
#15 /tmp/storage/runtime/compiled_templates/8c/8c816d6bf3fce4137510d4f4cba284fe.php(74): Twig\Template->yieldBlock('content', Array, Array)
#16 /app/user/vendor/twig/twig/src/Template.php(437): __TwigTemplate_4dd96cfd80394db323389a1c195304db->block_body(Array, Array)
#17 /tmp/storage/runtime/compiled_templates/9e/9e5ae5d3f2c09075ca6e26bb307279b7.php(110): Twig\Template->yieldBlock('body', Array, Array)
#18 /app/user/vendor/twig/twig/src/Template.php(393): __TwigTemplate_1e01f0397bc6d46659f3b1c10289259f->doDisplay(Array, Array)
#19 /tmp/storage/runtime/compiled_templates/8c/8c816d6bf3fce4137510d4f4cba284fe.php(51): Twig\Template->yield(Array, Array)
#20 /app/user/vendor/twig/twig/src/Template.php(393): __TwigTemplate_4dd96cfd80394db323389a1c195304db->doDisplay(Array, Array)
#21 /tmp/storage/runtime/compiled_templates/09/099aea1042afbf0528769d5e1edddcf0.php(60): Twig\Template->yield(Array, Array)
#22 /app/user/vendor/twig/twig/src/Template.php(393): __TwigTemplate_93328f652dcac9c5548e1a67014fa730->doDisplay(Array, Array)
#23 /tmp/storage/runtime/compiled_templates/74/7467b9498591f30fc4b77cb15f1f8206.php(51): Twig\Template->yield(Array, Array)
#24 /app/user/vendor/twig/twig/src/Template.php(393): __TwigTemplate_27a50d20589a8df34435d0a71d4438c4->doDisplay(Array, Array)
#25 /app/user/vendor/twig/twig/src/Template.php(349): Twig\Template->yield(Array, Array)
#26 /app/user/vendor/twig/twig/src/Template.php(364): Twig\Template->display(Array)
#27 /app/user/vendor/twig/twig/src/TemplateWrapper.php(35): Twig\Template->render(Array)
#28 /app/user/vendor/twig/twig/src/Environment.php(306): Twig\TemplateWrapper->render(Array)
#29 /app/user/vendor/craftcms/cms/src/web/View.php(501): Twig\Environment->render('index', Array)
#30 /app/user/vendor/craftcms/cms/src/web/View.php(554): craft\web\View->renderTemplate('index', Array)
#31 /app/user/vendor/craftcms/cms/src/web/TemplateResponseFormatter.php(57): craft\web\View->renderPageTemplate('index', Array, 'site')
#32 /app/user/vendor/yiisoft/yii2/web/Response.php(1109): craft\web\TemplateResponseFormatter->format(Object(craft\web\Response))
#33 /app/user/vendor/craftcms/cms/src/web/Response.php(339): yii\web\Response->prepare()
#34 /app/user/vendor/yiisoft/yii2/web/Response.php(340): craft\web\Response->prepare()
#35 /app/user/vendor/yiisoft/yii2/base/Application.php(390): yii\web\Response->send()
#36 /app/user/web/index.php(18): yii\base\Application->run()
#37 {main}
```

### Steps to reproduce

Not sure but it seems to be related to the element cache

### Craft CMS version

5.4.7.1

### Plugin version

2.1.0

### Multi-site?

Yes

**Comment by engram-design** [comment]
Should be fixed in [2.1.1](https://github.com/verbb/hyper/releases/tag/2.1.1)

**Comment by boboldehampsink** [comment]
Thanks!
